Ireland_Rosen_Chapters_4_5_6.tex

```diff
@@ -2517,7 +2513,7 @@
 \begin{align*}
 \prod_{k=1}^{(p-1)/2}((1+t)^{2k-1} - (1+t)^{p-(2k-1)})& = \prod_{k=1}^{(p-1)/2} \left((1+ (2k-1) t ) - (1+(p-(2k-1)) t +t^2u(t)\right)\\
 &=\prod_{k=1}^{(p-1)/2} \left((4k-2-p)t + t^2v(t)\right)\\
-&=t^{(p-1)/2} \prod_{k=1}^{(p-1)/2} (4k-2-p) + t^{(p-1)/2} w(t),
+&=t^{(p-1)/2} \left(\prod_{k=1}^{(p-1)/2} (4k-2-p)\right) + t^{(p+1)/2} w(t),
```

Comment:
The parentheses make it clear that the term involving $w(t)$ is outside the product. Also, it's important to note that this term is a multiple of $t^{(p+1)/2}$, hence does _not_ contribute to the coefficient of $t^{(t-1)/2}$.